src/test/java/rx/internal/operators/OperatorIgnoreElementsTest.java

```diff
@@ -92,7 +92,7 @@ public void call(Integer t) {
                     }
                 })
                 //
-                .ignoreElements()
+                .<Integer>ignoreElements()
```

Comment:
I came across a semver for Java library awhile ago that could take two jars and hint if it was major, minor or patch change.